hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3MultipartUploadCompleteRequest.java

```diff
@@ -497,12 +536,16 @@ private static String failureMessage(String volume, String bucket,
         volume + " bucket: " + bucket + " key: " + keyName;
   }
 
+  @SuppressWarnings("parameternumber")
   private void updateCache(OMMetadataManager omMetadataManager,
+      String dbBucketKey, OmBucketInfo omBucketInfo,
```

Comment:
NIT: we can void passing an extra parameter here if we use the omBucketInfo to generate the dbBucketKey here.

Reply:
In this method, omBucketInfo can be null to represent the update having no size change, which is set a few lines ago. 
```java
        if (usedBytesDiff != 0) {
          omBucketInfo.incrUsedBytes(usedBytesDiff);
          dbBucketKey = omMetadataManager.getBucketKey(
              omBucketInfo.getVolumeName(), omBucketInfo.getBucketName());
        } else {
          // If no bucket size changed, prevent from updating bucket object
          omBucketInfo = null;
        }
```
This nullability is also used in `S3MultipartUploadCompleteRequest` to avoid the change in the batch (avoiding one update in RocksDB).


To make my intention clearer, I added a `@Nullable` annotation and added a few lines of comment to ensure why it can be null.